src/main.ts

```diff
@@ -11,6 +12,11 @@ export async function run (): Promise<void> {
     return
   }
 
+  const serviceAccountKey = core.getInput('gcp_sa_key')
+  if (serviceAccountKey) {
+    await gcp.setupGcpSa(serviceAccountKey)
```

Comment:
```suggestion
    await gcp.setupServiceAccount(serviceAccountKey)
```
It stutter if we use `gcp` twice. Hence `gcp.setupServiceAccount`.